models/index.js

```diff
@@ -10,12 +10,7 @@ const db = {};
 
 let sequelize;
 if (config.use_env_variable) {
-  sequelize = new Sequelize(
-    config.database,
-    config.username,
-    process.env[config.use_env_variable],
-    config
-  );
+  sequelize = new Sequelize(process.env[config.use_env_variable]);
```

Comment:
For production, config only has use_env_variable. database and username are all set up automatically when using JAWSDB_URL